tools/circle_plus_gen/example/train_tparam.json

```diff
@@ -0,0 +1,19 @@
+{
+  "optimizer": {
+      "type": "adam",
+      "args": {
+          "learningRate": 0.01,
+          "beta1": 0.9,
+          "beta2": 0.999,
+          "epsilon": 1e-07
+      }
+  },
+  "loss": {
+      "type": "categorical crossentropy",
```

Comment:
(question)
Where can I get the type string?

Reply:
There is no document for type string rules. I'll add a document and more example files for details. 

-------------------------------

For now, you could check the allowable type string only in code level. 

https://github.com/Samsung/ONE/blob/9fe820c3e05eedec50edf4039a66dfb26d2fb322/tools/circle_plus_gen/lib/utils.py#L38